<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AESCipherSystem</class>
 <widget class="QMainWindow" name="AESCipherSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Help">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QToolButton" name="FileChooseBut">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>33</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilePathLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>651</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the File to be encrypted/decrypted :-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Encrypt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Decrypt">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the Directory to save to :-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="About">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QToolButton" name="SaveToolButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>170</y>
      <width>33</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SavePathLnEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>651</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="SamePathCheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to same path as original file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Done">
    <property name="geometry">
     <rect>
      <x>687</x>
      <y>500</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AESKey">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>431</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="AESiv">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>431</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the Key for AES Cipher : -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the initialization vector : -</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="Version"/>
    <addaction name="separator"/>
    <addaction name="AboutTool"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="Version">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="AboutTool">
   <property name="text">
    <string>About Tool</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
